app/controllers/games_controller.rb

```diff
@@ -24,34 +27,34 @@ def show
     end
   end
 
-  def update
-    game = Game.find_by(id: params[:id])
-    if game
-      action = case params[:task]
-               when 'cardMove'
-                 CardMover.new(game, params[:card], params[:action_id])
-               when 'incRound'
-                 IncrementRound.new(game, params[:player][:id], params[:action_id])
-               when 'changeTokens'
-                 ChangeToken.new(game, params[:token], params[:action_id])
-               else
-                 render json: { error: "Unknown action: #{params[:taskl]}", next_action: game.next_action }, status: 500
-                 return
-               end
-      action.call
-      if action.error.present?
-        render json: { error: action.error, next_action: game.next_action }, status: 500
-      else
-        render json: GameRender.new(game, current_user).call
-      end
-    else
-      redirect_to root_path
-    end
-  end
+  # def update
+  #   game = Game.find_by(id: params[:id])
+  #   if game
+  #     action = case params[:task]
+  #              when 'cardMove'
+  #                CardMover.new(game, params[:card], params[:action_id])
+  #              when 'incRound'
+  #                IncrementRound.new(game, params[:player][:id], params[:action_id])
+  #              when 'changeTokens'
+  #                ChangeToken.new(game, params[:token], params[:action_id])
+  #              else
+  #                render json: { error: "Unknown action: #{params[:taskl]}", next_action: game.next_action }, status: 500
+  #                return
+  #              end
+  #     action.call
+  #     if action.error.present?
+  #       render json: { error: action.error, next_action: game.next_action }, status: 500
+  #     else
+  #       render json: GameRender.new(game, current_user).call
+  #     end
+  #   else
+  #     redirect_to root_path
+  #   end
+  # end
 
   def join
     game = Game.find_by(id: params[:id])
-    if (game_player_id = game&.join(current_user))
+    if (game_player_id = game&.join(player_name))
       cookies["game_player_id_#{game.id}"] = game_player_id
```

Comment:
why do this? well the user_id is their unique key, so If I expose that in the game data it would be possible for other people to spoof that player (change there user_id to another players - phil)

This is a simple solution to avoid that without much effort... long term this should all be replaced with some sort of actual auth process (most likely devise) - but that is not currently required and is a low priority compared to ACTUALLY making the game work..

Once the game works I will be a lot more interested in User Auth... also user game tracking, etc... but only if the game actually first, as I have done auth at every job ever... I have not previously built a game